src/message.rs

```diff
@@ -10,10 +11,13 @@ use super::config::Rtd;
 pub fn handle_message(
     client: &IrcClient, message: Message, rtd: &Rtd, db: &Database
 ) {
+
```

Comment:
Extra blank line?

Reply:
Hmm, indeed, i had originally put the regex build here, think i omitted tidying up properly